src/background.js

```diff
@@ -3,23 +3,52 @@ import ky from 'ky'
 const prettifyJson = obj => JSON.stringify(obj, null, 2)
 
 // schedules
-chrome.alarms.create('cache-shortcuts', { delayInMinutes: 1, periodInMinutes: 1 })
+chrome.alarms.create('cache-shortcuts', { periodInMinutes: 1 })
```

Comment:
Run once each minute, w/o delay

**looking to the future:** will probably need to trigger the following alarm listener from the content-script.  Best practice seems to discourage persistent background pages.